contributor-docs/release-guide.md

```diff
@@ -877,7 +877,8 @@ pull request, just correct it on the spot and the vote can continue as-is.
 ### Run validation tests
 
 The community is responsible for performing validation, but as release manager
-you are expected to contribute as well.
+you are expected to contribute as well. Please see the [RC Testing Guide](https://github.com/apache/beam/blob/master/contributor-docs/rc-testing-guide.md)
```

Comment:
We should add this link to the templated email as well instead of the link to the blog - https://github.com/apache/beam/blob/c9c89fe7d5f88b004635bd941725f6213c7d1cdc/contributor-docs/release-guide.md?plain=1#L836 (though ideally more prominently)